packages/block-library/src/home/edit.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText, useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+
+const preventDefault = ( event ) => event.preventDefault();
+
+export default function HomeEdit( { attributes, setAttributes, clientId } ) {
+	const blockProps = useBlockProps();
+	const { homeUrl } = useSelect(
+		( select ) => {
+			const {
+				getUnstableBase, //site index
+			} = select( coreStore );
+			return {
+				homeUrl: getUnstableBase()?.home,
+			};
+		},
+		[ clientId ]
+	);
+
+	const { label } = attributes;
+
+	return (
+		<li { ...blockProps }>
```

Comment:
> Naming: is core/home okay, or do we prefer something else?

At the moment the implementation isn't very general, so Home doesn't seem to be a great fit.

Reply:
I didn't show it in this branch, but we could update the behavior such that the li wrapper is only added when the parent is a navigation block. This is similar to what the PageList does

Reply:
Interesting idea to offer an automatic wrapper element.